src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct

```diff
@@ -245,7 +251,7 @@
     <GuidSymbol name="guidToolsGroup" value="{C0D88179-5D25-4982-BFE6-EC5FD59AC103}">
       <IDSymbol name="idLibraryPackageManager" value="0x100" />
       <IDSymbol name="idToolsGroup" value="0x200" />
-      <IDSymobl name="idGeneralSettings" value="0x300" />
+      <IDSymbol name="idGeneralSettings" value="0x300" />
```

Comment:
what does this do 😆

Reply:
Or what wasn't it doing?

Reply:
Good question, I don't know what it wasn't doing.